README.md

```diff
@@ -188,8 +189,42 @@ Fine tune on 10% of Pascal:
 GPUS_PER_NODE=2 ./tools/run_dist_launch.sh 2 ./configs/DETReg_fine_tune_10pct_pascal.sh --batch_size 4 --epochs 200 --lr_drop 150
 ```
 
+### Pretraining on ImageNet with pseudo labels from MDETR / MDef-DETR proposals
 
+[comment]: <> (The MDETR proposals generated on ImageNet100 is saved in: )
+The command for pretraining DETReg on 8 GPUs on ImageNet100 is as following:
```

Comment:
Let's change the commands to `training on 4 gpus` instead as we trained all DETReg models on 4 GPUs